modules/rotating-certificate-authority

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// security/certificate_types_descriptions/etcd-certificates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rotating-certificate-authority_{context}"]
+=== Rotating the etcd certificate authority
+
+Rotate the `etcd` certificate authority (CA) before the certificate expires.
+
+.Procedure
+
+. Verify the remaining lifetime of the new signer certificate:
+
+[source,terminal]
+----
+$ oc get secret -n openshift-etcd etcd-signer -ojsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}'
+----
+
+. Recreate the signer, if the remaining lifetime is close to the current date, by deleting the signer and wait for the status pod rollout:
+
+[source,terminal]
+----
+$ oc delete secret -n openshift-etcd etcd-signer
+----
++
+[source,terminal]
+----
+$ oc wait --for=condition=Progressing=False --timeout=15m clusteroperator/etcd
+----
+
+. After the CA rotates, you can switch the original CA with the new one.
```

Comment:
the CA doesn't rotate (yet), etcd restarts :)

Reply:
:-) Thanks Thomas!